src/Executables/CombineH5/CombineH5.cpp

```diff
@@ -19,26 +20,31 @@
 // main module we just have it be empty
 extern "C" void CkRegisterMainModule(void) {}
 
-// Compares source archives of each file against the 0th volume file. Returns
-// true if each source archive matches the 0th one
-bool check_src_files(const std::vector<std::string>& file_names) {
-  h5::H5File<h5::AccessType::ReadWrite> initial_file(file_names[0], true);
-  const auto& src_archive_object_initial =
-      initial_file.get<h5::SourceArchive>("/src");
-  const std::vector<char>& src_tar_initial =
-      src_archive_object_initial.get_archive();
-  for (size_t i = 1; i < file_names.size(); ++i) {
-    h5::H5File<h5::AccessType::ReadWrite> comparison_file(file_names[i], true);
-    const auto& src_archive_object_compare =
-        comparison_file.get<h5::SourceArchive>("/src");
-    const std::vector<char>& src_tar_compare =
-        src_archive_object_compare.get_archive();
-
-    if (src_tar_initial != src_tar_compare) {
-      return false;
-    }
+// Returns all the observation_ids stored in the volume files. Assumes all
+// volume files have the same observation ids
+std::vector<size_t> return_observation_ids(const std::string& file_prefix,
```

Comment:
since basically all functions return something, what do you think about `get_observation_ids`? that's the more common phrasing